projects/liveart/index.js

```diff
@@ -1,25 +1,44 @@
 const sdk = require('@defillama/sdk')
 
-async function computeMarketCap(api, asset) {
-  const abi = 'function latestRoundData() view returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound)'
-  const { answer: price } = await api.call({ abi, target: asset.priceFeed })
-  const totalSupply = await api.call({ abi: 'erc20:totalSupply', target: asset.assetId })
-  const marketCap = (Number(totalSupply) / 10 ** 18) * (Number(price) / 10 ** 8)
-  sdk.util.sumSingleBalance(api.getBalances(), 'coingecko:usd-coin', marketCap)
+async function computeMarketCap(api, assets, token) {
+  const priceFeedAbi = 'function latestRoundData() view returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound)'
+  const assetList = Array.isArray(assets) ? assets : [assets]
+
+  await Promise.all(assetList.map(async (asset) => {
```

Comment:
instead of using a .map() here, can we batch the calls into multicalls with api.multiCall() ?